great_expectations/expectations/core/expect_column_values_to_be_null.py

```diff
@@ -84,21 +85,26 @@ class ExpectColumnValuesToBeNull(ColumnMapExpectation):
     map_metric = "column_values.null"
     args_keys = ("column",)
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
-        """
-        Validates that a configuration has been set, and sets a configuration if it has yet to be set. Ensures that
-        necessary configuration arguments have been provided for the validation of the expectation.
+        """Validates the configuration of an Expectation.
```

Comment:
can you please include a description of what's required for the configuration to be valid?

Reply:
@donaldheppner no logic has been implemented, so nothing is required at the Expectation-level. I guess if we want we can look though all of the super-class logic and update every Expectation with that?